pages/_app.js

```diff
@@ -135,7 +171,7 @@ const pages = [
   },
   {
     pathname: '/css-in-js',
-    title: 'CSS in JS (experimental)',
+    title: 'CSS in JS (alpha)',
```

Comment:
Would like to add this to lab too. We do have a disclaimer but this way it's more obvious.

Reply:
@eps1lon I believe we could kill the lab package by following React approach. They prefix unstable module with `unstable_`. They also warn about the non semver behavior in the documentation.

> WARNING: "unstable_" APIs may change in any version without respecting semver

https://gist.github.com/bvaughn/8de925562903afd2e7a12554adcdda16

Reply:
I like that. As it stands right now I have the feeling people miss the fact that they are alpha.

Reply:
This system thing is more challenging than anticipated. I'm going to extract the unrelated changes.

Reply:
Fixed.